src/main/kotlin/Navigation/NavigationProject.kt

```diff
@@ -0,0 +1,234 @@
+package Navigation
+
+import Archive.Archive
+import Note.TheNote
+import java.util.Scanner
+
+class NavigationProject {
+    val map: MutableMap<Int, MutableList<String>> = mutableMapOf()
+    val archiveSet: MutableSet<Archive> = mutableSetOf()
+    val scaner = Scanner(System.`in`)
+    val text = TextOutputToConsole()
+
+    fun navigationStart() {
+        println("\nВы запустили приложение \"АРХИВЫ\"")
+
+        while (true) {
+            text.printListMenuConclusion()
+            printListArchiveMenu()
+
+            when (navigationScanNumberOutput()) {
+                0 -> {
+                    println("----------   ВСЕГО ДОБРОГО   ----------")
```

Comment:
🍏 Чтобы после работы твоей программы цвет вывода консоли сбросился, нужно после последнего вывода добавить сброс цвета. Ты можешь сохранить комбинацию отдельно в константу или enum с остальными цветами
```suggestion
                    println("----------   ВСЕГО ДОБРОГО   ----------\u001b[0m")
```

Reply:
+